lib/ProjectContent.ts

```diff
@@ -153,40 +153,17 @@ export class ProjectContentV2 extends ProjectContent {
   }
 
   public async findCard(issueOrPR: IssueOrPR): Promise<Card> {
-    // FIXME: Pagination - replace "first:100" with paging. There will be more than 100 issues
-    console.log("FIXME pagination");
-    const {
-      organization: {
-        projectV2: {
-          items
+    let hasNextPage = true;
+    let endCursor = "";
+    while (hasNextPage) {
+      const items = await this.loadPaginatedCardItems(endCursor);
```

Comment:
Is there a better way to decompose this other than the parenthesis wrap? That's very confusing.
```
;({nodes, pageInfo: hasNextPage, endCursor} = await this.loadPaginatedCardItems(endCursor));
```

Reply:
Wouldn't using `const` work?

```js
const {nodes, pageInfo: { hasNextPage, endCursor } } = await this.loadPaginatedCardItems(endCursor);
```

Reply:
No, because `while (hasNextPage)` needs to read that variable outside it's scope.